docs/audit-log-filter-json.md

```diff
@@ -0,0 +1,92 @@
+# Audit Log Filter format - JSON
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The JSON format uses JSON arrays that contain properties as a key-value pair. These arrays represent an event in the audit. Some pairs are listed in every audit record. The audit record type determines if other key-value pairs are listed. The order of the pairs within an audit record is not guaranteed. The value description may be truncated.
```

Comment:
"uses JSON arrays" - may be a bit misleading as for me. Each file contains one top level array which in turn contains JSON objects with key-value pairs. 
"These arrays represent" - > These objects represent